routes/coverage.js

```diff
@@ -89,11 +94,20 @@ var saveCoverage = function ( hash, coverage, coverageType, options, callback )
                 return compare.hash === hash;
             } );
 
-            priorHash = indexOfCurrent === -1
-                ? hashes[ 0 ]
-                : hashes.length > indexOfCurrent + 1
-                    ? hashes[ indexOfCurrent + 1 ]
-                    : null;
+            var getPriorHashOnMaster = function ( indexOfCurrent )
```

Comment:
Right now there is no branch handling and the assumed behavior is basically that the PR is to master. There is an open issue to support branches at https://github.com/vokal/cvr-view/issues/40 so when that get's resolved that should correct for both these comments.